### DISCUSSION TITLE
2-hand touch support is not working with the S pen

### DISCUSSION THREAD

**Discussion by UnveiledGolem** [ppy/osu#22771] [Q&A]
### Type

Game behaviour

### Bug description

Samsung recently came with a new phone (s23 ultra) which has a pen (S pen) connected to it. After some testing I figured out that the S pen doesn't work with the 2 hand touch support that was implemented a couple of weeks ago.

### Version

2023.301.0

### Logs

[database.log](https://github.com/ppy/osu/files/10873670/database.log)
[network.log](https://github.com/ppy/osu/files/10873671/network.log)
[performance.log](https://github.com/ppy/osu/files/10873673/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10873674/runtime.log)

**Comment by agatemosu** [comment]
Because when the S Pen is being used, One UI disables the touch.

**Reply by ThangChoBoc** [reply]
Well .. is there a way to turn off the palm detection?
